Yosemite/YosemiteTests/Tools/ShippingSettings/StorageShippingSettingsServiceTests.swift

```diff
@@ -10,7 +10,7 @@ final class StorageShippingSettingsServiceTests: XCTestCase {
 
     override func setUp() {
         super.setUp()
-        storage = CoreDataManager(name: "WooCommerce")
+        storage = CoreDataManager(name: "WooCommerce", crashLogger: MockCrashLogger())
```

Comment:
If you're gonna touch this again later, would you mind adding a `tearDown` for `storage`? 😁 Low hanging fruit!

Reply:
updated https://github.com/woocommerce/woocommerce-ios/pull/2366/commits/c4634acfcfd2b821fb832d8284a57489da952ffc since this is easy to add 🙂